pkg/internal/controller/controller.go

```diff
@@ -303,7 +305,12 @@ func (c *Controller) reconcileHandler(ctx context.Context, obj interface{}) {
 		return
 	}
 
-	log := c.Log.WithValues("name", req.Name, "namespace", req.Namespace)
+	var log logr.Logger
+	if c.LoggerCustomizer != nil {
+		log = c.LoggerCustomizer(c.Log, req)
```

Comment:
We could also call c.LoggerCustomizer after l.312 instead of making it an if/else

I think I would prefer the current variant as there are no request specific k/v pairs in c.Log this way